src/components/SummaryCard/SummaryCardContainer/SummaryCardContainer.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file Summary card container.
+ * @copyright IBM Security 2019
+ */
+
+import React, { useState } from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as summaryCardNamespace } from '../SummaryCard';
+import { namespace as summaryCardSelectNamespace } from '../SummaryCardSelect/SummaryCardSelect';
+
+const getSummaryCard = (summaryCards, selectedId) =>
+  summaryCards.filter(({ id }) => id === selectedId);
+
+const resetSelectedSummaryCards = state => state([]);
+
+const SummaryCardContainer = ({ render, summaryCards }) => {
```

Comment:
I'm wondering why we need `SummaryCardContainer`? Do you think this could be added to `SummaryCard` instead?

Reply:
Good question! This closely mimics the `DataTable` API - https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/DataTable/DataTable.js

We need the `SummaryCardContainer` to control the batch action state of all the summary cards and batch actions components, as well as return the calculated props and output for each of the sub-components, for example, `getBatchActionProps`, `getSelectionProps`, and `selectedSummaryCards`.

Admittedly the naming seems off but I wanted to follow the `DataTable` example as closely as possible - I'm entirely open to suggestions here!